anyway/parsers/cbs/executor.py

```diff
@@ -839,7 +839,7 @@ def delete_invalid_entries(batch_size):
 
     marker_ids_to_delete = (
         db.session.query(AccidentMarker.id)
-        .filter(or_((AccidentMarker.longitude == None), (AccidentMarker.latitude == None)))
+        .filter(or_((AccidentMarker.longitude is None), (AccidentMarker.latitude is None)))
```

Comment:
Notice it's not a pythonic operation but SqlAlchemy, from reading this message on Stackoverflow(below) i don't believe it will work

https://stackoverflow.com/a/5632224/13771969